wiki/en/Running-a-Server.md

```diff
@@ -36,86 +37,108 @@ However, various problems can also arise when setting up Servers - especially wh
+* Jamulus offers limited IPv6 support that you can turn on for a Client or Server from the command line.
 
-## Server Types
+---
 
-You can run your Server in different ways (either at home or on a 3rd party host):
+## Installation
 
-### 1. Registered
+Most people run Jamulus on a 3rd party/cloud host as a "headless" Server (no video display or keyboard) on **hardware without audio** running Linux. You can also run a Server in a [**desktop environment**](#servers-on-the-desktop).
 
-Your Server will be listed in a Directory. By default, Jamulus has a list of built-in Directories that Clients can connect to. If you register with one of these, anyone can then discover and connect to your Server. You can also have your Server listed on a Custom Directory, if that better meets your needs.
+To run a headless server on Linux, the following steps assume you are familiar with the command line and Debian/Ubuntu or similar distribution that uses systemd. 
 
-<figure>
-	<img src="{% include img/en-screenshots/diagram-reg-server.inc %}" loading="lazy" alt="Diagram of connections between Clients within a Jamulus Registered Server">
-<figcaption>How Registered Servers work</figcaption>
-</figure>
+If you want to run a Server on a **Raspberry Pi** (or a different armhf/arm64 debian-based device), you will need to download the [latest armhf .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }}) or [latest arm64 .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-arm64 }}) depending on your CPU - not the default `amd64` ones for use on Intel/AMD machines.
+
+1. Download the [latest headless (amd64) .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }}) (or for Raspberry Pi etc. as above)
+1. Make sure you have a current list of standard packages: `sudo apt update`
+1. Install the Jamulus package for your architecture, for example: 
+
+	`sudo apt install ./{{ site.download_file_names.deb-headless }}`
+
+1. Enable the headless Server process: 
+
+	`sudo systemctl enable jamulus-headless`
+
+1. Add your desired [command line options](Running-a-Server#configuration) to the `ExecStart` line in the systemd service file (by default you will be running an Unregistered Server):
 
-### 2. Unregistered
+	`sudo systemctl edit --full jamulus-headless`
 
-This is the default when starting a Server for the first time. Unregistered Servers are not listed by Directories, so only musicians who know your Server's address to will be able to connect to it. This is useful because Jamulus does not let you control who can connect to a Server.
+1. Reload the systemd files and restart the headless Server:
+
+	`sudo systemctl daemon-reload && systemctl restart jamulus-headless` 
+
+_To upgrade your Server, just repeat the steps above._
+
+_To amend your Server configuration, just repeat the last two steps above._
+
+---
+
+## Server Types
```

Comment:
```suggestion
## Server modes
```

It is one Server in one of 3 modes. But I think there are internal page hyperlinks to this title.